src/libs/OptionsListUtils.js

```diff
@@ -36,7 +36,7 @@ function getPersonalDetailsForLogins(logins, personalDetails) {
             personalDetail = {
                 login,
                 displayName: login,
-                avatarURL: getDefaultAvatar(login),
+                avatar: getDefaultAvatar(login),
```

Comment:
One thing to note with this rename is that we'll have duplicate keys until the users session is invalidated. 
![image](https://user-images.githubusercontent.com/16747822/109726197-1f2fb400-7b67-11eb-8494-6e4d230ed269.png)